alf/config_util.py

```diff
@@ -60,7 +60,7 @@ def config(prefix_or_dict,
            mutable=True,
            raise_if_used=True,
            sole_init=False,
-           override=False,
+           override_sole_init=False,
```

Comment:
please also add tests for override_sole_init.

Reply:
`alf.override_config` is simply a wrapper for `alf.config(override_sole_init=True)`.